src/meson.build

```diff
@@ -44,3 +44,10 @@ pkg_mod.generate(libraries : lc3lib,
                  name : 'liblc3',
                  filebase : 'lc3',
                  description : 'LC3 codec library')
+
+#Declare dependency
+liblc3_dep = declare_dependency(
+	link_with:lc3lib,
+	include_directories:inc)
```

Comment:
Formatting nit: there should be a space after `:`.

Reply:
Done